scripts/sync_readme.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -x
+
+SCRIPTS_DIR="$( cd "$( dirname "$0" )" >/dev/null 2>&1 && pwd )"
+DIR=$(cd $SCRIPTS_DIR; cd ..; pwd)
+ROOT_README=$DIR/README.md
+MASTER_SITE_README=./site/docs/master/README.md
```

Comment:
This should be `$DIR/site/...` so that it's independent of where it's being run from.